src/plugins/artifact/editor/adapters/githubPluginAdapter.js

```diff
@@ -79,26 +78,27 @@ const adapter: PluginAdapter<NodePayload> = {
     function extractAuthorTitle(node: Node<AuthorNodePayload>) {
       return node.payload.login;
     }
-    type TypedNodeToStringExtractor = <T: $Values<NodeTypes>>(
-      T
-    ) => (node: Node<$ElementType<T, "payload">>) => string;
-    const extractors: $Exact<$ObjMap<NodeTypes, TypedNodeToStringExtractor>> = {
-      ISSUE: extractIssueOrPrTitle,
-      PULL_REQUEST: extractIssueOrPrTitle,
-      COMMENT: (node) => extractCommentTitle("comment", node),
-      PULL_REQUEST_REVIEW_COMMENT: (node) =>
-        extractCommentTitle("review comment", node),
-      PULL_REQUEST_REVIEW: extractPRReviewTitle,
-      USER: extractAuthorTitle,
-      ORGANIZATION: extractAuthorTitle,
-      BOT: extractAuthorTitle,
-    };
-    function fallbackAccessor(node: Node<NodePayload>) {
-      throw new Error(`unknown node type: ${node.address.type}`);
+    const anyNode: Node<any> = node;
+    const type: NodeType = (node.address.type: any);
```

Comment:
why the any-cast in `(node.address.type: any)`?

would it be cleaner to write `anyNode.address.type`?